pkg/iac-providers/helm/v3/load-dir.go

```diff
@@ -0,0 +1,289 @@
+/*
+    Copyright (C) 2020 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package helmv3
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	k8sv1 "github.com/accurics/terrascan/pkg/iac-providers/kubernetes/v1"
+
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
+	"github.com/accurics/terrascan/pkg/utils"
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
+	"helm.sh/helm/v3/pkg/engine"
+)
+
+var (
+	errSkipTestDir       = fmt.Errorf("skipping test directory")
+	errNoHelmChartsFound = fmt.Errorf("no helm charts found")
+	errBadChartName      = fmt.Errorf("bad chart name in Chart.yaml")
+	errBadChartVersion   = fmt.Errorf("bad chart version in Chart.yaml")
+)
+
+// LoadIacDir loads all helm charts under the specified directory
+func (h *HelmV3) LoadIacDir(absRootDir string) (output.AllResourceConfigs, error) {
+
+	allResourcesConfig := make(map[string][]output.ResourceConfig)
+
+	// find all Chart.yaml files within the specified directory structure
+	fileMap, err := utils.FindFilesBySuffix(absRootDir, []string{helmChartFilename})
```

Comment:
Can the helm char file also have `.yml` extenstion? In that case, `FindFilesBySuffix` will return an empty fileMap

Reply:
The helm chart file only supports Chart.yaml as far as I can tell. I added .yml at first because rancher uses it, but figured we would probably need to support rancher in a separate iac-type, so I reverted the change.

Reply:
Confirmed this is indeed the case in the helm source (only looks for "Chart.yaml")